ArduCopter/Copter.h

```diff
@@ -690,6 +696,9 @@ class Copter : public AP_Vehicle {
     // crash_check.cpp
     void crash_check();
     void thrust_loss_check();
+    void yaw_imbalance_check();
+    LowPassFilterFloat yaw_I_filt = LowPassFilterFloat(0.05f);
```

Comment:
we don't normally put the variables here.. can we move them somewhere else?

Reply:
Open to better ideas. The existing crash check functions use statics.